### ISSUE TITLE
Unable to pull docker images from Sudan

### ISSUE THREAD

**Issue by MomenNano** [docker/hub-feedback#2103]
Docker Hub is still blocked in Sudan, even though Sudan was lifted off the sanctions and restriction list by the US.

If you visit hub.docker.com from Sudan you will get:
"403 Forbidden 
Since Docker is a US company, we must comply with US export control regulations. In an effort to comply with these, we now block all IP addresses that are located in Cuba, Iran, North Korea, Republic of Crimea, Sudan, and Syria. If you are not in one of these cities, countries, or regions and are blocked, please reach out to https://hub.docker.com/support/contact/"

Is there any reason other that the restrictions by the US that makes you from updating the block list?
If there is no reason, when can we expect it to be accessible in Sudan?

**Comment by monzersmiledev** [comment]
Same here

**Comment by mohdule** [comment]
Sudan is no longer sanctioned nor a US embargoed nation,
While Atlassian, GitLab, Google already updated their policies and unblocked us, I don't really understand why it's that long for Docker to catch up, It's really frustrating 😔.

**Comment by nimir** [comment]
On 5/1/2021 I've contacted Docker support and opened a ticket (No 28824)

I quote from the support team response at that time:

```
Thanks for contacting Docker Support.

Our Legal and Engineering teams are currently reviewing our policies concerning the Sudan sanctions. I do not have an ETA at this time. Our legal team is waiting for some additional documentation, but if/when a change is approved, the changes will be made by the engineering team relatively quickly.

Apologize for the inconvenience.
```

More than 4 months since this promise and still no update + I asked for an update on Aprile/23 and got no reply.

In these difficult times, it is understandable that generally most support teams work with less capacity and are jammed but I think this issue isn't treated with the level of importance it deserves.

**Comment by the94air** [comment]
Many of us have been texting you guys about this, we still haven't seen any changes nor promises regarding the situation. Developers and companies based in Sudan are in need of no obstacles using Docker, just like many around the world. Kindly, let us know why is it taking much longer. Are we ever going to be off listed? Because it's getting very frustrating. Don't you think we deserve more answers?! Thank you.

**Comment by nebuk89** [comment]
hey all, thanks for your patience. Our legal team is checking we are good :) please keep following this ticket for updates!

**Comment by HajMo** [comment]
@nebuk89 Is there any update?

**Comment by MomenNano** [comment]
It has been more than 10 days, but we didn't see any update on this yet. 😕

**Comment by mohdule** [comment]
😢

**Comment by MomenNano** [comment]
@nebuk89 any update?

**Comment by mohdule** [comment]
This is really sad, i ain't going to lie
😔😭

**Comment by HajMo** [comment]
@nebuk89 any update?

**Comment by MuizMahdi** [comment]
@nebuk89 I guess we're given a low priority, maybe even the lowest

**Comment by MomenNano** [comment]
This shouldn't take that much IMO, all other major companies unblock us within the first month after the US removed us from the embargoed list.
You said you will keep us updated but we don't see anything until know. please prioritize this so it can be done.

**Comment by the94air** [comment]
Looks like the Docker team have had other plans looking into this issue and decided to keep us hanging. I really wish this could have been fixed differently with a direct answer, "no!" 😐 This is a very unprofessional attitude. I really expected more than this.

**Comment by MomenNano** [comment]
104 days without any update or action!

**Comment by nimir** [comment]
Hi folks,

I just wanted to share that I've reached out to Docker legal team today with an official complaint about this.

Let aside the unjustified delay of over 10 months for a simple policy update task that is causing a major effect: "denial of service for all Sudanese people"; The lack of feedback and the poor communication we received here is even worse and simply unacceptable. I am personally disappointed and didn't expect it from Docker team.

Anyway, I will keep following up with the legal team and keep you posted.

**Comment by ingshtrom** [comment]
Sorry for the slow updates. We should have updated sooner that we were continuing to work on it behind the scenes. Unfortunately, the legal side of things are complex and it required time to ensure that everything was in order so that we didn't open up the company to unnecessary hardship or legal trouble.

With that said, **we got the go-ahead from our legal team to unblock Sudan and have done so this morning!** 

We appreciate your expressed interest and involvement in the community. Please continue to report any problems in this thread or open a support ticket.

Thank you,
Alex

**Comment by MomenNano** [comment]
Thanks very much for the effort!

**Comment by Moerning** [comment]
What about Iran? should developers always reserve to other methods other than simple modern web-development?
